test/auth.test.js

```diff
@@ -26,18 +29,24 @@ function mockApiResponse(options = {}) {
   return new window.Response(body, responseOptions);
 }
 
-let test = {};
 describe('Auth', () => {
-  beforeEach(() => { test.fetch = sinon.stub(window, 'fetch'); });
-  afterEach(() => test.fetch.restore());
+  let test = {};
+
+  beforeEach(() => {
+    test.fetch = sinon.stub(window, 'fetch');
+  });
+
+  afterEach(async() => {
```

Comment:
nit: this doesn't need to be `async` if we're not using `await` in the body

Reply:
Good catch, leftover artifact from an earlier version of this.